disnake/guild.py

```diff
@@ -832,6 +837,19 @@ def public_updates_channel(self) -> Optional[TextChannel]:
         channel_id = self._public_updates_channel_id
         return channel_id and self._channels.get(channel_id)  # type: ignore
 
+    @property
+    def safety_alerts_channel(self) -> Optional[TextChannel]:
+        """Optional[:class:`TextChannel`]: Return's the guild's channel where admins and
```

Comment:
```suggestion
        """Optional[:class:`TextChannel`]: Returns the guild's channel where admins and
```
feel free to fix the other two occurrences as well, though you don't have to :p

Reply:
0925d3568b45092d894f46a2d29ed447a9b033c7